onnxscript/rewriter/ort_fusions/bias_gelu.py

```diff
@@ -6,17 +6,33 @@
 
 
 class BiasGeluFusion(pattern.RewriteRuleClassBase):
+    def __init__(
+        self,
+        name,
+        *,
+        has_contrib_op: bool,
```

Comment:
```suggestion
        contrib_op: bool,
```

could you add a docstring (google style) to the class to describe this option?